custom_components/google_home/api.py

```diff
@@ -227,3 +229,89 @@ async def update_google_devices_information(self) -> List[GoogleHomeDevice]:
             ]
         )
         return coordinator_data
+
+    async def delete_timer_or_alarm(
+        self, device: GoogleHomeDevice, item_to_delete: str
+    ) -> None:
+        """Deletes a timer or alarm,
+        can also delete multiple if a list is provided (Not implemented yet)."""
+
+        url = self.create_url(str(device.ip_address), PORT, API_ENDPOINT_DELETE)
+
+        # We need to remove charset=UTF-8 or else it will return a 400 BAD request.
+        # I think this is because of the character "/" in the id string.
+        HEADERS.update(
+            {
+                HEADER_CAST_LOCAL_AUTH: str(device.auth_token),
+                HEADER_CONTENT_TYPE: "application/json",
+            }
+        )
+
+        data = {"ids": [item_to_delete]}
+
+        item_type = item_to_delete[:5]
+
+        _LOGGER.debug(
+            "Deleting %s from Google Home device %s - %s - " "Raw data: %s",
+            item_type,
+            device.name,
+            url,
+            data,
+        )
+
+        try:
+            async with self._session.post(
+                url, json=data, headers=HEADERS, timeout=TIMEOUT
+            ) as response:
+                if response.status == HTTP_OK:
+                    resp = await response.json()
+                    if resp:
+                        if "success" in resp:
+                            if resp["success"]:
+                                _LOGGER.debug(
+                                    "Successfully deleted %s for %s",
+                                    item_type,
+                                    device.name,
+                                )
+                            else:
+                                _LOGGER.debug(
+                                    "Couldn't delete %s for %s - %s",
+                                    item_type,
+                                    device.name,
+                                    resp,
+                                )
+                        else:
+                            _LOGGER.error(
+                                (
+                                    "Failed to get a confirmation that the %s"
+                                    "was deleted for device %s. "
+                                    "Received = %s"
+                                ),
+                                item_type,
+                                device.name,
+                                resp,
+                            )
+                else:
+                    _LOGGER.error(
+                        "Failed to delete %s for %s, API returned" " %d: %s",
+                        item_type,
+                        device.name,
+                        response.status,
+                        response,
+                    )
+        except ClientConnectorError:
+            _LOGGER.debug(
```

Comment:
This probably worth an actual warning.
```suggestion
            _LOGGER.warning(
```